src/Apps/Collect/CollectApp.tsx

```diff
@@ -0,0 +1,35 @@
+import { Serif } from "@artsy/palette"
+import React, { Component } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { Box } from "Styleguide/Elements/Box"
+import { Flex } from "Styleguide/Elements/Flex"
+import { ArtworkGridFragmentContainer as ArtworkGrid } from "./Components/ArtworkGrid"
+
+export interface CollectAppProps {
+  name: string
+  query?: any
+}
+
+export class CollectApp extends Component<CollectAppProps> {
+  render() {
+    return (
+      <Flex flexDirection="column">
+        <Box my={2}>
```

Comment:
@l2succes - thanks again for bringing `styled-system` to the table and setting all this in motion -- IMO it's the greatest UI win since React 👍